lib/tasks/metadata.rake

```diff
@@ -0,0 +1,43 @@
+namespace :fromthepage do
+  desc 'Refresh metadata'
+  task :refresh_metadata, [:id, :type] => :environment do |_task, args|
+    id = args[:id]
+    type = args[:type]
+
+    raise ArgumentError, 'Usage: rake fromthepage:refresh_metadata[id_value,type_value]' if id.nil? || type.nil?
+
+    case type
+    when 'collection'
+      p "Refreshing metadata for works in collection #{id}"
+      object = Collection.find_by(id: id)
+      all_works = object.works
+    when 'document_set'
+      p "Refreshing metadata for works in document_sets #{id}"
+      object = DocumentSet.find_by(id: id)
+      all_works = object.works
+    when 'work'
+      p "Refreshing metadata for work #{id}"
+      all_works = Work.where(id: id)
+    else
+      raise ArgumentError, 'Type can only be collection, document_set, or work' if id.nil?
+    end
+
+    all_works.in_batches(of: 1000).each do |works|
+      works.each do |w|
+        next unless w.sc_manifest
+
+        begin
+          manifest = JSON.parse URI.open(w.sc_manifest.at_id).read
+          _old = JSON.parse w.original_metadata
+          new =  manifest['metadata']
+          updated << w.slug
+          w.original_metadata = new.to_json
+          w.save!
+          p "Updated metadata for work #{w.id}"
```

Comment:
Let's move the preceding code to a method on `Work` -- that should help us test it better and also make it more available from the console.